src/routes/boot.ts

```diff
@@ -328,6 +336,49 @@ const getUser = (con: DataSource, userId: string): Promise<User> =>
     ],
   });
 
+const getMarketingCta = async (
+  con: DataSource,
+  log: FastifyBaseLogger,
+  userId: string,
+) => {
+  if (!userId) {
+    log.info('no userId provided when fetching marketing cta');
+    return null;
+  }
+
+  let marketingCta: MarketingCta | null = null;
+  const redisKey = generateStorageKey(
+    StorageTopic.Boot,
+    'marketing_cta',
```

Comment:
Not sure it exists, but don't we have enum for storage keys? (maybe it's only on apps though)

Reply:
Yeah maybe, it's not for you, but something we should have done a while ago 😂

Reply:
Added it just to have done it